webui/react/src/pages/Settings/Settings.tsx

```diff
@@ -1,13 +1,15 @@
 import { Tabs } from 'antd';
```

Comment:
What's the story behind this change?

Reply:
This page used to not sync tab information to url, I changed it to align with the other places in code base, should we have a separate PR for it?

Reply:
It's fine since the change is small but you should call that out when you make the PR, preferably by commenting on the relevant file in the PR discussion.

Reply:
Got it, thanks!